### DISCUSSION TITLE
Author Section CCD Specification

### DISCUSSION THREAD

**Discussion by JJWest12** [nextgenhealthcare/connect#5496] [Q&A]
I am trying to create a "perfect" CCD parsing tool in that it can parse through any "normal" CCD possible. What I have been able to come up with is that each clinical document can have 1 or more authors. Each author has to have 1 assignedAuthor, and that assignedAuthor has to have a representedOrganization. So essentially, each author tag has to have a representedOrganization.

What I am struggling to now find out, and I have been searching the internet for a bit, is what the details are for the name and addr inside the representedOrganization. Is there always 1 name? Can there be more than 1? Same issues with the addr. If anyone knows more or can point me to something that I didn't find on my own I would greatly appreciate it!

**Comment by siddharthsharma9258** [comment]
I think you would need the help for this website - https://site.healthit.gov/sandbox-ccda/ccda-validator
I used it many years ago to validate my CCD documents.

**Reply by JJWest12** [reply]
I don't so much need validation for a case by case basis, I need to know what a CCD "should" have in it

**Reply by pacmano1** [reply]
You should go read the specs yourself.   You are going to encounter hundreds of questions like the one you posted here if you want to create a generic CDA parser.   

Go look at bluebutton, MDHT, and other projects to see how they do stuff.
